lib/helpers.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Helper functions
+
+# Extract item or items matching .name, including deduplication
+# $1: item name, prepended or not with deduplication mark
+array_from_name() {
+  item_name="$(echo "$1" | sed "s/$DEDUP_MARK //")"
+  echo "$ITEMS" | jq -r ". | map(select(.name == \"$item_name\"))"
```

Comment:
Right here, you can also get back *identities*, which is something that can not return a password. We should filter this statement to only return *logins*

Reply:
Looking at my data more carefully I noticed there is a type field:
```
1 -> login
2 -> note
3 -> card
4 -> identity (you were probably mentioning this one)
```
This could be useful in the future, so I'd rather go for something like:
``` bash
TYPE_LOGIN=1
...
echo "$ITEMS" | jq -r ". | map(select((.name == \"$item_name\") and (.type == $TYPE_LOGIN)))"
```

Reply:
Good catch!